src/app/components/blocks/Sidebar.Links.tsx

```diff
@@ -4,16 +4,22 @@ import { ReactComponent as OverviewIcon } from "app/icons/Overview.svg";
 import { ReactComponent as ReceiveIcon } from "app/icons/Receive.svg";
 import { ReactComponent as SendIcon } from "app/icons/Send.svg";
 import { ReactComponent as SwapIcon } from "app/icons/SwapIcon.svg";
-import { ReactComponent as AppsIcon } from "app/icons/Apps.svg";
+// import { ReactComponent as AppsIcon } from "app/icons/Apps.svg";
 import { ReactComponent as ContactsIcon } from "app/icons/Contacts.svg";
 import { ReactComponent as WalletsIcon } from "app/icons/Wallets.svg";
 import { ReactComponent as SettingsIcon } from "app/icons/Settings.svg";
 import { ReactComponent as SupportIcon } from "app/icons/Support.svg";
+import { ReactComponent as ActivityIcon } from "app/icons/ActivityIcon.svg";
 import * as SupportAlert from "app/components/elements/SupportAlert";
 import { useDialog } from "app/hooks/dialog";
+import { useSetAtom } from "jotai";
+import { activityModalAtom } from "app/atoms";
+import { useActivityBadge } from "app/hooks";
 
 const useSidebarLinks = () => {
   const { alert } = useDialog();
+  const setActivityOpened = useSetAtom(activityModalAtom);
+  const showActivityBadge = useActivityBadge();
```

Comment:
"showActivityBadge" sounds like action/function. Better to call boolean vars like "activityBadgeDisplayed" | "activityBadgeShowed"